cli/src/commands/extensions/api.rs

```diff
@@ -349,27 +353,35 @@ fn run_sandboxed(process: Process) -> Result<ProcessOutput> {
             };
 
             for path in process.exceptions.read.sandbox_paths().iter() {
-                permissions::add_exception(&mut birdcage, Exception::Read(PathBuf::from(path)))
+                let path = dirs::expand_home_path(path, &home_dir);
+                permissions::add_exception(&mut birdcage, Exception::Read(path))
                     .map_err(into_ioerr)?;
             }
             for path in process.exceptions.write.sandbox_paths().iter() {
-                permissions::add_exception(&mut birdcage, Exception::Write(PathBuf::from(path)))
+                let path = dirs::expand_home_path(path, &home_dir);
+                permissions::add_exception(&mut birdcage, Exception::Write(path))
                     .map_err(into_ioerr)?;
             }
             for path in process.exceptions.run.sandbox_paths().iter() {
-                let absolute_path = permissions::resolve_bin_path(path);
+                let absolute_path =
+                    permissions::resolve_bin_path(dirs::expand_home_path(path, &home_dir));
```

Comment:
We've got a linebreak here anyway, I think it would be nice for consistency to extract the home path resolution into an intermediate value. That way it's much more consistent and thus it's going to be easy to spot where changes need to be made if we need to do so in the future.

Reply:
Agreed